The app is not installed

I have encountered some other issues, could you help me take a look? @Aslemammad 

Actually, we have already installed it. 

Related to https://github.com/web-infra-dev/rspack/pull/7757

```bash
Run pnpm run test:pkg

> monorepo@1.0.3 test:pkg /home/runner/work/rspack/rspack
> pnpm exec pkg-pr-new publish --compact --pnpm './npm/*' './crates/node_binding' './packages/rspack'

{"url":"/check","statusCode":404,"statusMessage":"Not Found","message":"The app https://github.com/apps/pkg-pr-new is not installed on web-infra-dev/rspack."}
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

--- a/packages/backend/server/routes/check.post.ts
+++ b/packages/backend/server/routes/check.post.ts
@@ -6,18 +6,31 @@ export default eventHandler(async (event) => {
 
   const app = useOctokitApp(event);
 
+  let authenticated = false;
+
   try {
     await app.octokit.request("GET /repos/{owner}/{repo}/installation", {
       owner,
       repo,
     });
-  } catch {
+    authenticated = true;
+  } catch {}
+
+  try {
+    await app.octokit.request("GET /orgs/{org}/installation", {
+      org: owner,
+    });
+    authenticated = true;
+  } catch {}
+
+  if (!authenticated) {
     throw createError({
       statusCode: 404,
       fatal: true,
       message: `The app https://github.com/apps/pkg-pr-new is not installed on ${owner}/${repo}.`,
     });
   }
+
   const workflowData = await workflowsBucket.getItem(key);
 
   if (!workflowData) {

